cmd/soroban-rpc/internal/db/sqlmigrations/03_ledgers_index.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+
+-- creating an index in the ledgers table on the sequence number.
+CREATE INDEX index_lcm_sequence ON ledger_close_meta(sequence);
```

Comment:
isn't the `sequence` column a primary key for the `ledger_close_meta` table? If so, I'm don't think there's any benefit in having this index, see https://stackoverflow.com/questions/3379292/is-an-index-needed-for-a-primary-key-in-sqlite

Reply:
Really good point! It should be easy to figure out by running the benchmark with and without the index

Reply:
Yeah will run it with and without to check

Reply:
Its the same without the index too.